code-of-conduct-enforcement/cpg-onboarding.md

```diff
@@ -12,7 +12,7 @@ The behaviors that informed this ban with clear steps to avoid in future are des
 
 * Violation 1  - [CPG exact wording] , ways to avoid
 * Violation 2  - [CPG exact wording] , ways to avoid
-* Violation 3  - [CPG exact wording] , aways to avoid
+* Violation 3  - [CPG exact wording] , ways to avoid
 
 ## Reinstatement Description 
 Describe what access is being granted, and if any restrictions remain.
```

Comment:
They will be allowed to talk to customers that need help. Lead in meeting for the company. Be servers that help on the community channel to solve problems amongst the community so the people don't get out of hand. They will be able to show clientele how to reach more support if they do not have the resources to help them themselves.